util/csvresponse.py

```diff
@@ -48,18 +51,64 @@ def to_csv_files(self, path):
         file_paths = self._create_files(base_path)
         return json.dumps({"code": 200, "message": str(file_paths)}, indent=2)
 
+    def _create_filter_list(self,keys):
+        ''' generates a set of parameters to filter (remove) from the package
```

Comment:
Please follow the existing style for function comment blocks (see CvsGenerator, e.g.).

Reply:
Will do.